appveyor.yml

```diff
@@ -1,9 +1,10 @@
 environment:
+  RUSTFLAGS: -Zunstable-options -Ctarget-feature=+crt-static
   matrix:
   - TARGET: i686-pc-windows-msvc
     BUILD_MSI: 1
   - TARGET: i686-pc-windows-gnu
-    MINGW_URL: https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/4.9.2/threads-win32/dwarf/i686-4.9.2-release-win32-dwarf-rt_v4-rev4.7z/download
+    MINGW_URL: https://s3.amazonaws.com/rust-lang-ci
```

Comment:
This was a bit of an unrelated cleanup, I switched to the url that we're using for Rust and a few other projects, and AWS is generally much more reliable than sourceforge